src/java.xml/share/legal/xhtml11.md

```diff
@@ -47,7 +47,7 @@ The notice is:
 "Copyright © 2023 W3C®. This software or document includes material copied from
 or derived from [title and URI of the W3C document]."
 
-Disclaimers §anchor
```

Comment:
Did that come from an upstream file ?

Reply:
No, it is copy/pasted from a textual rendering of the html file specified in the URL above. This is what you get if you naïvely select the text in Firefox and press Ctrl-C. The `§anchor` part is not rendered on screen.